artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -557,6 +592,25 @@ public Object run(ActionContext context) throws Exception {
          filters.put("${global-max-section}", readTextFile(ETC_GLOBAL_MAX_SPECIFIED_TXT, filters));
       }
 
+      if (jdbc) {
+         if (jdbcURL == null) {
+            jdbcURL = "jdbc:derby:" + getInstance().getAbsolutePath() + "/data/derby/db;create=true";
```

Comment:
Should this be left null as well rather than defaulting to Derby?

Reply:
Nope.. .lets not complicate things even further? :)

The config will be defaulted to Derby if no parameters.. then the user can install derby. it's a nice experience out of box. easy to install. easy to run.

Reply:
@jbertram I want to keep this as is.. with some default to be used.